modules/oem/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ set(O3D3XX_OEM_ETC_DIR "/opt/oem/etc/o3d3xx_oem"
 option(BUILD_TESTS "Build unit tests" ON)
 option(BUILD_EXE_JITTER "Build o3d3xx-oem-jitter" ON)
 option(INSTALL_HEADERS "Add headers to install package" OFF)
+option(INSTALL_SUPPLEMENTARY "Add supplementary files to install package" ON)
```

Comment:
This was necessary to cleanly separate building the library for the toolchain and for the device.